page_formatter.rb

```diff
@@ -0,0 +1,15 @@
+# all 3 of these requires are necessary
+require 'rubygems'
+require 'bundler/setup'
+require 'mechanize'
+
+class PageFormatter
+
+	def self.get_page_from(url)
```

Comment:
This class is not necessary. `get_page_from` can be a method in `DiceWebScraper`